backend/routes/srcs/in/abroadSharing/main.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const router_auth = express.Router()
+const router = express.Router()
+
+router_auth.post('/addAbroadSharing', require('./addAbroadSharing'))
+router.get('/getAbroadSharing', require('./getAbroadSharing'))
+router_auth.delete('/deleteAbroadSharing', require('./deleteAbroadSharing'))
+router_auth.patch('/updateAbroadSharing', require('./updateAbroadSharing'))
+module.exports = { router, router_auth }
```

Comment:
split router and router_auth st. it is more clear which are for router and which are for router_auth